deploy/Dockerfile

```diff
@@ -27,13 +27,15 @@ ENV CSI_ENDPOINT=unix:///csi/csi.sock   \
 RUN apk add --no-cache \
     cryptsetup \
     e2fsprogs e2fsprogs-extra \
-    xfsprogs \
-    xfsprogs-extra \
     lsblk \
     blkid \
     kmod \
     $EXTRA_PACKAGES
 
+RUN apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/v3.14/main  \
```

Comment:
@husam-lb what has triggered this issue ? and do you know why this is not an issue on the internal lb-csi ?

Reply:
mounting xfs formatted volume was failing 
see LBM1-21995.